content/workshops/jboss-migration-toolkit/lab4-mavenize-app.md

```diff
@@ -0,0 +1,131 @@
+{{% /alert %}}
+
+## Step 2. Review the Maven Project Structure
+The `/path/to/output/mavenized/APPLICATION_NAME/` directory will contain the following items:
+
+* A root POM file. This is the pom.xml file at the top-level directory.
+* A BOM file. This is the POM file in the directory ending with -bom.
+* One or more application POM files. Each module has its POM file in a directory named after the archive. 
+
+The example `jee-example-app-1.0.0.ear` application is an EAR archive that contains a WAR and several JARs. There is a separate directory created for each of these artifacts. Below is the Maven project structure created for this application. 
+
+~~~~
+/path/to/output/mavenized/jee-example-app/
+    jee-example-app-bom/pom.xml
+    jee-example-app-ear/pom.xml
+    jee-example-services2-jar/pom.xml
+    jee-example-services-jar/pom.xml
+    jee-example-web-war/pom.xml
+    pom.xml
+~~~~
+
+Review each of the generated files and customize as appropriate for your project. To learn more about Maven POM files, see the Introduction to the POM section of the [Apache Maven documentation][1]. 
+
+### Root POM File
+The root POM file for the `jee-example-app-1.0.0.ear` application can be found at `/path/to/output/mavenized/jee-example-app/pom.xml`. This file identifies the directories for all of the project modules.
+
+The following modules are listed in the root POM for the example `jee-example-app-1.0.0.ear` application. 
+
+~~~~
+<modules>
+  <module>jee-example-app-bom</module>
+  <module>jee-example-services2-jar</module>
+  <module>jee-example-services-jar</module>
+  <module>jee-example-web-war</module>
+  <module>jee-example-app-ear</module>
+</modules>
+~~~~
+
+{{% alert info %}}
+Be sure to reorder the list of modules if necessary so that they are listed in an appropriate build order for your project.
+{{% /alert %}}
+
+### BOM File
+The Bill of Materials (BOM) file is generated in the directory ending in -bom. For the example jee-example-app-1.0.0.ear application, the BOM file can be found at `/path/to/output/mavenized/jee-example-app/jee-example-app-bom/pom.xml`. The purpose of this BOM is to have the versions of third-party dependencies used by the project defined in one place. For more information on using a BOM, see the [Introduction to the Dependency Mechanism][2] section of the Apache Maven documentation. 
+
+The following dependencies are listed in the BOM for the example `jee-example-app-1.0.0.ear` application 
+
+~~~~
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.6</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
+      <version>2.5</version>
+    </dependency>
+  </dependencies>
+</dependencyManagement>
+~~~~
+
+### BOM File
```

Comment:
You repeat the *BOM File* heading here. Is that intentional or do both sets of content belong under the heading by the same name above?